src/tsserver/webServer.ts

```diff
@@ -81,6 +93,19 @@ namespace ts.server {
         }
     }
 
+
+    function createVirtualFileSystem() {
+        Debug.assert(sys === undefined);
+        // TODO: I don't think I need the XMLHttpRequest code since vfs will get its files from messages sent by the owner
+        // ...but this means I may not need the webSession-copied code in startVirtualFileSystemSession
+        const vfshost = TestFSWithWatch.createVirtualServerHost([]);

```

Comment:
We should probably rename `TestFSWithWatch`.

Reply:
How does it relate to `server.createWebSystem`?  Does one delegate to the other?

Reply:
server.createWebSystem should create a ServerHost that delegates to a vfs ServerHost for filesystem operations.